lib/fathead/django_built_ins/parse.py

```diff
@@ -0,0 +1,177 @@
+from bs4 import BeautifulSoup
+
+DJANGO_HOME = 'https://www.djangoproject.com/'
+DJANGO_DOC_URL = 'https://docs.djangoproject.com/en/1.9/ref/templates/builtins/'
```

Comment:
Here are the steps as I'm seeing it:
1) Create a file that contains configuration parameters. Text file, JSON, something else? Thoughts on what would be the best approach here?
2) The `fetch.sh` script would read that file for the django version value and pass it through to the (to be made) `'fetch_html.sh` script
3) The `parse.sh` script will also read that file for the django version and pass it through to the `parse.py` command.

Does this sound reasonable?